labelbox/schema/project.py

```diff
@@ -431,8 +430,6 @@ def export_v2(self,
                         _params.get('data_row_details', False),
                     "includeProjectDetails":
                         _params.get('project_details', False),
-                    "includeLabels":
```

Comment:
Shouldn't that be labelDetails instead?
https://github.com/Labelbox/intelligence/blob/develop/apps/api/src/exports-v2/export-data.graphql#L17